packages/components/src/internal/components/forms/detail/DetailDisplay.tsx

```diff
@@ -456,7 +457,7 @@ export function resolveDetailRenderer(column: QueryColumn): Renderer {
                 renderer = d => <LabelColorRenderer data={d} />;
                 break;
             case 'filecolumnrenderer':
-                renderer = d => <FileColumnRenderer data={d} col={column} />;
+                renderer = d => <FileColumnRenderer data={d} isFileLink={column.rangeURI === FILELINK_RANGE_URI} />;
```

Comment:
is `rangeURI === FILELINK_RANGE_URI` worth adding to the QueryColumn as a helper? something like "isFileColumn()"